clients/nym-client/src/clients/mod.rs

```diff
@@ -39,14 +42,17 @@ impl MixTrafficController {
     async fn run(mut rx: mpsc::UnboundedReceiver<MixMessage>) {
         let mix_client = mix_client::MixClient::new();
         while let Some(mix_message) = rx.next().await {
-            println!(
+            info!(
                 "[MIX TRAFFIC CONTROL] - got a mix_message for {:?}",
                 mix_message.0
             );
             let send_res = mix_client.send(mix_message.1, mix_message.0).await;
             match send_res {
-                Ok(_) => println!("We successfully sent the message!"),
-                Err(e) => eprintln!("We failed to send the message :( - {:?}", e),
+                Ok(_) => {
+                    print!(".");
+                    io::stdout().flush().ok().expect("Could not flush stdout");
```

Comment:
Do we need to do that?

Reply:
Yeah. My goal is to have ".", "..", "..." etc progressively print out as loop cover traffic is received, and then show "real" messages with actual content in them on their own line. 

As it turns out, if you are printing a small message like "." using `print!` in Rust, nothing shows up. This is because `print!` *does not auto-flush* (but `println!` does auto-flush). So that `flush` is needed to get the effect I'm looking for.